script/1_0_2current_en_script/Common/00_00.txt

```diff
@@ -10,43 +10,43 @@ What day was it?
 What was going on?
 My mind couldn't keep up.
 <Page4>
-When I put strength into my hands to try and get up, I felt a sharp pain in my body before my hands.
+When I put strength into my hands to try and get up, I felt a sharp pain in my body.
 I gasped for air like a fish flapping its mouth.
 <Page5>
 That feeling, it was as though fire was coming out of my eyes.
 As soon as I placed my hands on the firm bed, a searing pain cut deep into me.
 <Page6>
 "――― ―――"
-I took a deep breath and inspected my body while I was lying down.
+I took a deep breath and inspected my body while still lying down.
 <Page7>
 My hands were covered in bandages.
-My left leg was also wrapped in a white plaster.
+My left leg was also wrapped in white plaster.
 I felt an awful tightness around my forehead, so it was probably wrapped in bandages too.
 There was a large, thick adhesive bandage on my chest, covering up anything inconvenient.
 <Page8>
-There was a single tube inserted in my left arm.
+There was a single tube inserted into my left arm.
 I could vaguely understand that this was an IV drip.
 <Page9>
-I guessed it was because I was looking around half-asleep before waking up.
+Maybe because I was looking around half-asleep before waking up.
 <Page10>
 The darker room that smelled of medicine,
 the numerous tubes extending from the bandage on my chest,
-the scary man in the suit that came to visit,
+the scary man in the suit who came to visit,
 the sight of my sister being scolded by that man,
-and even the sight of the doctor frolicking like a child,
-all of which, I remembered as though it were a dream.
+and even the sight of the doctor frolicking like a child.
+I remembered them all, as though I saw them in a dream.
```

Comment:
I find that saying he 'saw them in a dream' implies it was something he saw in the past when it should just be Shiki being really groggy.